src/cbc_sdk/workload/vm_workloads_search.py

```diff
@@ -71,17 +95,104 @@ def _refresh(self):
+            config_file (str): The text of a config.ini file with a list of sensor properties to configure
+                               on installation.
 
         Returns:
-            str: The URL used to make requests for this object.
+            dict: A dict with two members, 'type' and 'code', indicating the status of the installation.
+
+        Raises:
+            NotImplementedError: Always, for BaseComputeResource.
+        """
+        raise NotImplementedError("Resource does not allow sensor installation")
+
+    @classmethod
+    def bulk_install(cls, cb, compute_resources, sensor_kit_types, config_file=None):
+        """
+        Install a sensor on a list of compute resources.
+
+        Args:
+            cb (BaseAPI): Reference to API object used to communicate with the server.
+            compute_resources (list): A list of ComputeResource objects used to specify compute resources to install
+                                      sensors on.
+            sensor_kit_types (list): A list of SensorKit objects used to specify sensor types to choose from
+                                     in installation.
+            config_file (str): The text of a config.ini file with a list of sensor properties to configure
+                               on installation.
+
+        Returns:
+            dict: A dict with two members, 'type' and 'code', indicating the status of the installation.
+
+        Raises:
+            NotImplementedError: Always, for BaseComputeResource.
+        """
+        raise NotImplementedError("Resource does not allow sensor installation")
+
+    @classmethod
+    def bulk_install_by_id(cls, cb, compute_resources, sensor_kit_types, config_file=None):
+        """
+        Install a sensor on a list of compute resources, specified by ID.
+
+        Args:
+            cb (BaseAPI): Reference to API object used to communicate with the server.
+            compute_resources (list): A list of dicts, each of which contains the keys 'vcenter_uuid' and
+                                      'compute_resource_id', specifying the compute resources to install sensors on.
+            sensor_kit_types (list): A list of SensorKit objects used to specify sensor types to choose from
+                                     in installation.
+            config_file (str): The text of a config.ini file with a list of sensor properties to configure
+                               on installation.
+
+        Returns:
+            dict: A dict with two members, 'type' and 'code', indicating the status of the installation.
+
+        Raises:
+            NotImplementedError: Always, for BaseComputeResource.
+        """
+        raise NotImplementedError("Resource does not allow sensor installation")
+
+
+class VCenterComputeResource(BaseComputeResource):
+    """Models a vCenter compute resource."""
+    def __init__(self, cb, model_unique_id, initial_data=None):
+        """
+        Initialize the AWSComputeResource object.
```

Comment:
Maybe this should be VCenterComputeResource.

Reply:
I'll go over those docs and make sure the types match.